src/components/DiffView/index.tsx

```diff
@@ -13,24 +22,81 @@ class DiffView extends React.Component<Props> {
     viewType: 'unified',
   };
 
-  render = () => {
+  renderHeader({ hunks }: DiffInfo) {
+    const { additions, deletions } = hunks.reduce(
+      (acc, hunk) => {
+        return {
+          additions:
+            acc.additions +
+            hunk.changes.filter((change) => change.isInsert).length,
+          deletions:
+            acc.deletions +
+            hunk.changes.filter((change) => change.isDelete).length,
+        };
+      },
+      { additions: 0, deletions: 0 },
+    );
+
+    return (
+      <div className={styles.header}>
+        <div className={styles.stats}>
+          <span className={styles.statsAdditions}>{`+++ ${additions}`}</span>
+          <span className={styles.statsDeletions}>{`--- ${deletions}`}</span>
+        </div>
+      </div>
+    );
+  }
+
+  renderHunk(hunk: HunkInfo, index: number) {
+    const components = [
+      <Hunk key={`hunk-${hunk.content}`} hunk={hunk} className={styles.hunk} />,
```

Comment:
Sticking the entire content into `key` seems a bit brutal. Is there an easier key value? I mean, I guess it works.

Reply:
That's what the `react-diff-view` lib does.